src/vivarium_csu_alzheimers/components/observers.py

```diff
@@ -47,6 +53,17 @@ def get_age_bins(builder: Builder) -> pd.DataFrame:
 
         return age_bins
 
+    def map_semester(self, pop: pd.DataFrame) -> pd.Series:
+        assert len(set(pop.event_time)) == 1, "All event_time values should be the same"
```

Comment:
Nit: `len(pop.event_time.unique()) == 1` might be more efficient, since it uses a pandas method to get the unique values rather than a python one.

Do we need this assertion though?